src/js/views/patients/sidebar/filters/filters-sidebar.hbs

```diff
@@ -2,12 +2,12 @@
   <div class="flex flex-align-center">
     <div class="flex-grow">
       <h3 class="sidebar__heading">
-        <span class="u-margin--r-8">{{far "sliders"}}</span>All Filters
+        <span class="u-margin--r-8">{{far "sliders"}}</span>{{ @intl.patients.sidebar.filters.filtersSidebarViews.allFiltersLabel }}
```

Comment:
I think this'd be `intl.patients.sidebar.filters.filtersSidebarTemplate.allFiltersLabel`

Reply:
Since this is now in the `HeaderView`, I moved this to `@intl.patients.sidebar.filters.filtersSidebarViews.headerView.allFiltersLabel`.